src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Figure implements Figureable {
```

Comment:
Figure class should be an abstract class since it's not meant to be instantiated directly.